hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -213,6 +565,13 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     console.log(current, pageSize);
   };
 
+  onTabChange = (activeKey: string) => {
+    // Fetch inactive pipelines if tab is switched to "Inactive"
+    if (activeKey === '2') {
+      // Fetch inactive pipelines in the future
+    }
+  };
+
```

Comment:
I don't understand this part and the comments. How do pipelines come here?

Reply:
Removed from Code